weecology/portalr
# Upkeep - improve messages + CI

~Tests won't pass because of #298~

Fixes #295 

Otherwise, use cli for messaging

* use testthat 3rd edition (use snapshot test for message validation + other lints)

* Update gh actions with `usethis::use_github_action()`

* Update pkgdown to use bootstrap 5 (adds a search bar)

* FIx a tidyselect deprecation warning
* Make format_*() functions redundant (no longer used in the package)

Happy to answer any questions you may have!

cli has the advantage of detecting automatically if it should do ansi output and fallbacks otherwise. This means that if you are using RStudio (or a terminal that supports links), test output is now clickable (as you can see, I was able to get rid of setup.R and teardown.R

R CMD CHECK passes cleanly locally ~except for the test failure~
